app/controllers/subtopics_controller.rb

```diff
@@ -1,57 +1,45 @@
 class SubtopicsController < ApplicationController
   before_filter { validate_slug_param(:topic_slug) }
   before_filter { validate_slug_param(:subtopic_slug) }
-  before_filter :send_404_if_not_found
   before_filter :set_slimmer_format
 
+  rescue_from GdsApi::HTTPNotFound, :with => :cacheable_404
```

Comment:
Should this be a `error_404` instead of a `cacheable_404`? I thought `cacheable_404` is for URLs that can't exist (malformed slugs).